callosum/lower/redis.py

```diff
@@ -237,6 +239,24 @@ class RedisStreamConnector(AbstractConnector):
             _redis.close()
             await _redis.wait_closed()
 
+    
+class TransportType(Enum):
```

Comment:
Currently transport type is only defined for Redis.
"COMMON" and "REDIS" seems not to reflect the intention you described in the comments.
How about just "UNIDRIECTIONAL" and "BIDIRECTIONAL"?
Or some other names such as "DISPATCH" (stream with load balanced consumers), "BROADCAST" (pub-sub), and "RPC" (bidirectional channel)?

Reply:
I renamed it to `DistributeRedisTransport` and `RPCStreamTransport`. All the Connection, Binder, Connector classes were renamed in the same fashion. I also separated them into two separate files. Since I decided to use different transports for two cases, it lead to using different connections as well. So everything got separated. That lead me into separating those things into two files: `lower/distribute_redis.py` and `lower/rpc_redis.py`.